packages/react-test-renderer/src/ReactTestRenderer.js

```diff
@@ -557,6 +557,10 @@ const ReactTestRendererFiber = {
   /* eslint-enable camelcase */
 
   unstable_setNowImplementation: setNowImplementation,
+
+  act<X>(callback: () => X): X {
+    return batchedUpdates(callback);
```

Comment:
Needs some way to flush passive effects.

Reply:
Whe I wrote my own hacks tests I cheated by just doing two interactiveUpdates batches :innocent: